lib/file-utils.go

```diff
@@ -102,6 +102,30 @@ func GetFromStorage(bucket string, file string, keyPath string) []byte {
 	return slurp
 }
 
+func GetFromStorageV2(bucket string, file string, keyPath string) ([]byte, error) {
+	var err error
+	log.Println("start GetFromStorageV2")
+	log.Println("File: " + file)
+	ctx := context.Background()
+	client, err := storage.NewClient(ctx)
+	if err != nil {
+		return nil, err
+	}
+	rc, err := client.Bucket(bucket).Object(file).NewReader(ctx)
+	if err != nil {
+		return nil, err
+	}
+	slurp, err := io.ReadAll(rc)
+	if err != nil {
+		return nil, err
+	}
+	err = rc.Close()
+	if err != nil {
+		return nil, err
+	}
+	return slurp, err
+}
+
```

Comment:
What's the difference between this and `GetFromStorageErr` function?

Reply:
Honestly, when I started refactoring the email function, I didn't realised there was `GetFromStorageErr`.
IMHO `GetFromStorageV2` has a better error handling so I would be inclined to keep this, at least for this PR. But if you decide otherwise the fix is a one minute work.

Reply:
Talking with Diogo, we have decided to remove  existent `GetFromStorageErr` and rename your `GetFromStorageV2` into `GetFromStorageErr`